cfgov/jinja2/v1/about-us/the-bureau/about-deputy-director/index.html

```diff
@@ -19,12 +19,16 @@
 
 	{% import 'molecules/social-media.html' as social_media %}
 
-    <div class="block block__flush-top" data-qa-hook="director-bio">
+    <div class="block block__flush-top"
+         data-qa-hook="director-bio">
         <h1 data-qa-hook="director-bio-title">
             Deputy Director David Silberman (Acting)
         </h1>
         <p class="lead-paragraph">
-            David Silberman currently serves as the Acting Deputy Director as well as the Associate Director of Research, Markets, and Regulations at the Consumer Financial Protection Bureau (CFPB).
+            David Silberman currently serves as the acting deputy director
```

Comment:
Question for @sarahsimpson09 and @tnferrell, this lead repeats what's in the paragraph below. Should it be removed from that paragraph?